### ISSUE TITLE
PULL_UP and PULL_DOWN constants don't match WiringPi?

### ISSUE THREAD

**Issue by andrewn** [nebrius/raspi-gpio#3]
I might be completely wrong, but are the `PULL_UP` and `PULL_DOWN` constants the wrong way round compared to `WiringPi`?

[raspi-io/index.js](https://github.com/nebrius/raspi-gpio/blob/master/index.js#L34-L36)

``` js
export const PULL_NONE = 0;
export const PULL_UP = 1;
export const PULL_DOWN = 2;
```

[wiringPi/wiringPi.h](http://git.drogon.net/?p=wiringPi;a=blob;f=wiringPi/wiringPi.h;h=dd110fa52ca08b40c3d42fe7e557341d2be2a35e;hb=HEAD#l51)

``` c
  52 // Pull up/down/none
  53 
  54 #define PUD_OFF                  0
  55 #define PUD_DOWN                 1
  56 #define PUD_UP                   2
```

This might explain the weird issue I'm seeing in nebrius/raspi-io#36 with PULL_DOWN/PULL_UP behaviour.

**Comment by nebrius** [comment]
Seems like it, nice catch! I don't use internal pullups in any of my projects myself, so I never tested it 😅

**Comment by nebrius** [comment]
Fixed with v2.1.0 of raspi-gpio, thanks again for the report!
